logger/log.go

```diff
@@ -0,0 +1,69 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package logger
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
```

Comment:
As we are adding the dependency, I think we need the LICENSE update and LICENSE check.

@kezhenxu94 Do you have a LICENSE check(besides header check)?

Reply:
> As we are adding the dependency, I think we need the LICENSE update and LICENSE check.
> 
> @kezhenxu94 Do you have a LICENSE check(besides header check)?

We only have source files' header checker, no dependencies' license checker

Reply:
OK, then we need a manual check. 

@EvanLjp We need two LICENSE(and Notice) files, one in the root, another in the `dist/`(also should be included in the binary tar).
1. LICENSE in the root folder, represents the license of the source codes. If you copied some codes from another repo(must be in Catalog A), then you need to add the description in the file.
1. LICENSE in the dist folder, represents the license of the binary, every dependency libs included in the binary level(runtime level), should be statemented there.

Read this for more details, https://www.apache.org/legal/resolved.html. As there are enough committers from 2 TLPs of the ASF, I think there should be more eyes on the LICENSE from day one.

Reply:
The files are there, I just share with you the principle. We need to be careful about the License. And you should update them every time, you importing.

Reply:
I think we need `go.mod`.